docs/guides/benchmarks.md

```diff
@@ -0,0 +1,175 @@
+# Benchmarks
```

Comment:
It should also be the first page in the user guide since it's so important (first entry in the mkdocs.yml user guide list), so I will wait for this to be merged and rebase before checking in the organization best practices guide.